api/external/applications/applications.proto

```diff
@@ -138,66 +249,100 @@ service ApplicationsService {
   };
 }
 
+// Request message for GetServices.
 message ServicesReq {
+  // Search bar filters to apply to services, in the format of fieldname:value.
   repeated string filter = 1;
+  // Pagination parameters to apply to the returns services list.
   common.query.Pagination pagination = 2;
+  // Sorting parameters to apply to the returns services list.
   common.query.Sorting sorting = 3;
 }
 
+// Request message for GetServicesDistinctValues.
 message ServicesDistinctValuesReq {
+  // Field name of service values to retrieve.
   string field_name = 1;
+  // Text of value to query, supports wildcards (* and ?).
   string query_fragment = 2;
+  // Filters to apply to values, in the format of fieldname:value.
   repeated string filter = 3;
 }
 
+// Request message for GetServicesBySG.
 message ServicesBySGReq {
+  // Service group ID of the services.
   string service_group_id = 1;
+  // Pagination parameters to apply to the returned list of services.
   common.query.Pagination pagination = 2;
+  // Sorting paramters to apply to the returned list of services.
   common.query.Sorting sorting = 3;
+  // Filters to apply to services, in the format of fieldname:value.
   repeated string filter = 5;
 }
 
+// Response message for GetServicesBySG.
 message ServicesBySGRes {
+  // Service group name.
   string group = 1;
+  // List of services.
   repeated Service services = 2;
+  // Health counts for services in the service group.
   HealthCounts services_health_counts = 3;
```

Comment:
The comment above this line will not appear in the UI. For singlular messages (ie, not `repeated`) in another message, it uses the comment from where the message is defined (in this case, line 444): https://github.com/chef/automate/pull/2286/files#diff-b0479669666449d6f06889b029243ad6R444

I added some explanation around this to the README: https://github.com/chef/automate/blob/master/components/automate-chef-io/README.md#message-documentation